helm/aws-pod-identity-webhook/templates/Deployment.yaml

```diff
@@ -63,6 +63,13 @@ spec:
             {{- with .Values.securityContext }}
               {{- . | toYaml | nindent 12 }}
             {{- end }}
+          resources:
```

Comment:
how about making this an helm variable? this would allow customizing in large clusters

Reply:
Done